src/include/stir/RelatedViewgrams.inl

```diff
@@ -79,10 +79,10 @@ int RelatedViewgrams<elemT>::get_basic_timing_pos_num() const
 }
 
 template <typename elemT>
-ViewSegmentNumbers RelatedViewgrams<elemT>::
+ViewSegmentTOFNumbers RelatedViewgrams<elemT>::
 get_basic_view_segment_num() const
 {
-  return ViewSegmentNumbers(get_basic_view_num(), get_basic_segment_num());
+  return ViewSegmentTOFNumbers(get_basic_view_num(), get_basic_segment_num(), get_basic_timing_pos_num());
```

Comment:
What is a `basic_timing_pos_num`? Need to check over logic here because this is probably not needed?

Reply:
Looks like `get_basic_view_num()` and `get_basic_segment_num()` are used in very few places. Arguably we could get rid- of them, but I see no strong need. They are just the `view/segment_num` of the "basic" element. So that extends nicely to `timing_pos_num` as well. (it's what will end up in the `basic_bin` and then transformed etc)

Reply:
so, here it's just the `tof` member